pontoon/contributors/templates/contributors/widgets/notifications_menu.html

```diff
@@ -47,8 +47,7 @@
 
         {% if description and (description.startswith("Unreviewed suggestions") or notification.verb == "has reviewed suggestions") %}
           <span class="description">{{ description|safe }}</span>
-
-          <p class="timeago">{{ notification.timesince() }} ago</p>
+          <time class="timeago" datetime="{{ notification.timestamp }}">{{ notification.timesince() }} ago</time>
```

Comment:
Note that the [inline-level](https://developer.mozilla.org/en-US/docs/Glossary/Inline-level_content) `<time>` element needs to have `display: block;` set in CSS in order to behave like a [block-level](https://developer.mozilla.org/en-US/docs/Glossary/Block-level_content) `<p>` element.